src/listen.js

```diff
@@ -45,6 +45,9 @@ module.exports = function(defaultFuncs, api, ctx) {
       case 'read_receipt':
         globalCallback(null, utils.formatReadReceipt(event));
         return true;
+      case 'read':
```

Comment:
Ok. No problem here.
Let's confirm there is any problem about the key naming issues.
If there's no problem, I'll give you a pull request.
Thanks.